static/styles/user_status.scss

```diff
@@ -10,7 +10,7 @@
     }
 
     input.user_status {
-        width: 420px;
+        width: 336px;
```

Comment:
What's the story with these width changes?  They seem not part of the other change, and not explained in the commit message.